restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
 import argparse
 import json
 import sys
+import locale
```

Comment:
We leverage the pep8 convention of grouping and alphabetizing core modules. Please add locale in alphabetical order with the rest of the modules.

https://www.python.org/dev/peps/pep-0008/#id17